interface/therapy_groups/index.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: amiel
+ * Date: 07/11/16
+ * Time: 12:01
+ */
+
```

Comment:
Need to use the security model flags here:
$fake_register_globals=false;
$sanitize_all_escapes=true;

(http://www.open-emr.org/wiki/index.php/Codebase_Security#SQL-Injection_and_Cross-Scripting_Prevention)